### PR TITLE
Example: add example ds18b20

### LINKED ISSUE #62: Epic ds18b20 Temperature sensor example required
This issue is referred to design a test to ds18b20. this allow know about the module ds18

### PR DESCRIPTION
**Pull request by eduazocar**
### Contribution description
Integrated solution to use the module ds18b20, apply the resources functions in ds18.h to get values of temperature in grades Celsius. 
  

### Testing procedure

Use in your code

ds18_init(&dev, &dev_params)  --> to initialize the device
ds18_get_temperature(&dev, &temp_value)  --> to get values of temperature in Celsius

### Issues/PRs references
fix #62

### CODE REVIEW

File: examples/esp32-wroom-32/test_ds18b20/main.c [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Eduardo Azócar <eduardo@turpialdev.com>
```
Comment (luisan00) [inline examples/esp32-wroom-32/test_ds18b20/main.c:2]: Wrong  copyright formar

File: examples/esp32-wroom-32/test_ds18b20/main.c [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+ */
+
+/**
+ * @file    main.c
+ * @author  Eduardo Azócar <eduardo@turpialdev.com>
+ * @brief   Example_of_ds18
+ * @version 0.1
+ * @date    2022-01-04
+ *
+ * @copyright Copyright (c) 2022
```
Comment (luisan00) [inline examples/esp32-wroom-32/test_ds18b20/main.c:24]: Same here

File: examples/esp32-wroom-32/test_ds18b20/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+### Go to the example folder
+
+```sh
+cd examples/esp-wroom32/test_ds18b20
+```
+
+### Compile your code and flash it
+
+```sh
+    make flash term
```
Comment (luisan00) [inline examples/esp32-wroom-32/test_ds18b20/README.md:26]: With no spaces before it, in consistent with the others documents 
```suggestion
make flash term
```

File: examples/esp32-wroom-32/test_ds18b20/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    ds18_params_t dev ={
+        .pin = sensor_pin,
+        .out_mode = sensor_mode_out // OUT_OPRN_DRAIN_PULL UP
+        .in_mode = sensor_mode_in
+    }
+```
+## Last steps
+Initialize the device
+```c
+    ds18_init(&dev,&dev_params); // Set the params to your sensor
```
Comment (luisan00) [inline examples/esp32-wroom-32/test_ds18b20/README.md:48]: Same here

File: examples/esp32-wroom-32/test_ds18b20/README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+```
+Check the temperature
+```c
+    ds18_get_temperature(&dev,&var_temperature); // var_temperature has to defined previously as int16_t variable
```
Comment (luisan00) [inline examples/esp32-wroom-32/test_ds18b20/README.md:52]: Here too

**Review by luisan00 — CHANGES_REQUESTED** [review]
Good job 👍 only some misprints

File: examples/esp32-wroom-32/test_ds18b20/main.c (hunk 2 above)
Comment (eduazocar) [reply examples/esp32-wroom-32/test_ds18b20/main.c:24]: sorry those last lines was generated by my vscode, i will fix it

File: examples/esp32-wroom-32/test_ds18b20/main.c (hunk 1 above)
Comment (eduazocar) [reply examples/esp32-wroom-32/test_ds18b20/main.c:2]: This is the same format in the coding convention (license and header copyright)

File: examples/esp32-wroom-32/test_ds18b20/Makefile [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+USEMODULE += saul_default
+# include the shell:
+USEMODULE += ds18
+USEMODULE += shell
+USEMODULE += shell_commands
+# additional modules for debugging:
+USEMODULE += ps
+USEMODULE += ztimer
+USEMODULE += xtimer
+
```
Comment (luisan00) [inline examples/esp32-wroom-32/test_ds18b20/Makefile:20]:
```suggestion
```

**Review by luisan00 — CHANGES_REQUESTED** [review]

**Review by luisan00 — APPROVED** [review]
let's go ahead :+1:

**Comment by eduazocar** [comment]
Ok it's time to squash

**Pull request merged**